Dangerfile

```diff
@@ -1,11 +1,8 @@
-# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
```

Comment:
I get removing this because it's not consistent, but I do like the thinking behind it.

Reply:
Yep! I think we just need a standard and then we can revisit.

Reply:
Perhaps we could encourage folks to use the Draft PR feature for these situations?

Reply:
That would be nice :). I am not sure what others options are on Draft PRs?